<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorsWidget</class>
 <widget class="QWidget" name="ColorsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Normals âˆ’:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QToolButton" name="toolButton_clip_plane_color">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="toolButton_wireframe_color">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="widget_background_color" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Background:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QWidget" name="widget_clip_plane_color" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="widget_wireframe_color" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Clip Plane:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QWidget" name="widget_normal_color_negative" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QToolButton" name="toolButton_normal_color_negative">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QWidget" name="widget_normal_color_positive" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="toolButton_background_color">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Normals +:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Wireframe:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QToolButton" name="toolButton_normal_color_positive">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QWidget" name="widget_dft_background_color" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>DFT bg:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="widget_dft_color" native="true">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>DFT:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="toolButton_dft_background_color">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="toolButton_dft_color">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>83</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
